frontend/src/commit-panel/commit/Form.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+interface FormProps {
+  commitMessage: string;
+  onCommitMessageChange(e: React.FormEvent): void;
+  onSubmit(e: React.FormEvent): void;
+  onCancelCommitClick(e: React.MouseEvent): void;
+}
+
+const Form: React.StatelessComponent<FormProps> = (
+  { commitMessage, onCommitMessageChange, onSubmit, onCancelCommitClick }) => (
```

Comment:
I would destruct the props in the body when they are this long to avoid line breaking